ldes-server-port-fetch-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/exceptionhandling/RestResponseEntityExceptionHandler.java

```diff
@@ -14,10 +16,12 @@
 @ControllerAdvice
 public class RestResponseEntityExceptionHandler
```

Comment:
This would be cleaner by adding handleException(Exception, WebRequest, HttpStatus) to reduce the repeated logic